modules/javafx.controls/src/main/java/javafx/scene/control/Skin.java

```diff
@@ -28,9 +28,22 @@
 import javafx.scene.Node;
 
 /**
- * Base class for defining the visual representation of user interface controls
+ * An interface for defining the visual representation of user interface controls
  * by defining a scene graph of nodes to represent the skin.
  * A user interface control is abstracted behind the {@link Skinnable} interface.
+ * <p>
+ * A Skin implementation should generally avoid modifying its control outside of
```

Comment:
I think this is a good start. As discussed elsewhere, additional implementation notes for Skin implementers would be helpful.

Reply:
added @ImplNote to install().